frontend/src/containers/Review/ReviewControl/ReviewControl.js

```diff
@@ -43,25 +101,25 @@ class ReviewControl extends Component {
             <div className="review-control">
                 <div className="review-control-page">
                     <div className="board">
+                        <div className="header">{this.props.mode ? "Edit Review For" : "Create Review For"}</div>
                         <div className="paper-spec">
                             <PaperSpec
                               id={this.state.paper.id}
                               title={this.state.paper.title}
+                              liked={this.state.paper.liked}
                               abstract={this.state.paper.abstract}
-                              date={this.state.paper.date}
-                              authors={this.state.paper.authors}
-                              keywords={this.state.paper.keywords}
-                              likeCount={this.state.paper.likeCount}
-                              reviewCount={this.state.paper.reviewCount}
+                              likeCount={this.state.likeCount}
+                              authors={this.state.authorNames}
```

Comment:
This part generates warnings and don't work as you thought! Did you run this?

Reply:
Also, you should pass `keywords`, too.

Reply:
I handle error (it is related to props, so other things worked well when I ran the program) and add keywords.

Reply:
@kosumin Yeah, as you can see on the screenshot I left, author names were not properly joined maybe because of it.